### ISSUE TITLE
untagged-<HASH> being set as Tag name

### ISSUE THREAD

**Issue by nchudleigh** [release-drafter/release-drafter#204]
New drafts have their tag name being set as untagged-<HASH>

Our version tags are set as `v1.1.1-staging` and `v1.1.1-production`.

Looking through the code I can see that the `semver` package is used to parse the version out of the `lastRelease`'s tag name.  This seems to work fine when I test locally.

![image](https://user-images.githubusercontent.com/5580824/57400600-d9d81080-71a1-11e9-9da2-a9fdf5ed2706.png)

Here are my relevant settings
```yml
# release-drafter.yml
name-template: "v$NEXT_PATCH_VERSION"
tag-template: "v$NEXT_PATCH_VERSION-staging"
```

My thought is perhaps the NEXT_PATCH_VERSION variable is causing an error when I append the `-staging` to it.  If this is the case, how do I properly separate the two.

**Comment by casz** [comment]
Test the code here: https://runkit.com/embed/d3dun8uc0h45
```
var semver = require("semver")
semver.coerce('v1.1.1-production', { includePrerelease: true })
```

Does not include preleases 😢

**Comment by nchudleigh** [comment]
Not sure I understand your runkit demo.  It doesnt seem your config is applied.  Even with includePrerelease as false, the coercion seems to work fine as I showed in my screenshot.

I am unable to replicate the issue, have made a test here, let me know if there is anything missing from it. It seems to pass the suite just fine.
https://github.com/toolmantim/release-drafter/compare/master...nchudleigh:test/pre-release

**Comment by casz** [comment]
[Read the docs for coerce, it will swallow prerelease](https://github.com/npm/node-semver#coercion)

**Comment by casz** [comment]
What you want it most likely #203

**Comment by valishah** [comment]
Hi, I'm facing a similar issue. Whenever I'm merging a new PR into master, It's been updated to release draft but **Tag version** seems to be overriding and keeping as **untagged-<hash>**. Though I'm changing it manually to the valid one (ex: 1.3.2) it's reverting back to the **untagged-<hash>** format on the subsequent PR merge into master. Is it intentional? That's how it supposed to be working?

**Comment by ostrolucky** [comment]
We have same problem. Any ideas?

**Comment by ajcann** [comment]
We're also experiencing this issue.

**Comment by KengoTODA** [comment]
Maybe it's not the same with your cases, but in my case, this problem is caused because I have no stable (non-draft) release in the repo.
For now, the configuration for `name` & `tag` aren't referenced in this condition:

https://github.com/release-drafter/release-drafter/blob/9740c7174dd5166604b2278117034ccc3fa29d97/lib/releases.js#L207-L212

Just for your reference. Thanks for sharing this great solution making our hacking comfortable! :+1:

**Comment by pdcmoreira** [comment]
> Maybe it's not the same with your cases, but in my case, this problem is caused because I have no stable (non-draft) release in the repo.

Manually creating a v1.0.0 release made subsequent release drafts (created by release-drafter) to get the correct tag, until I started using the development branch for releases. Now all drafts are untagged again.

**Comment by TimonVS** [comment]
I'm sorry for taking so long to reply to this issue. I think GitHub may have slightly changed their API since we released this feature as it seems that we should explicitly set the `tag_name` when updating a release, otherwise it'll be set to `untagged-<hash>`, the other possibility is that it never worked properly on our end 😅. 
In any case, I've created a PR to fix the issue (https://github.com/release-drafter/release-drafter/pull/367) which I'll be releasing soon.

**Comment by TimonVS** [comment]
The fix has been released in [v5.6.0](https://github.com/release-drafter/release-drafter/releases/tag/v5.6.0).
